services/auth.go

```diff
@@ -69,6 +69,14 @@ func GetSession(w http.ResponseWriter, r *http.Request) persistence.SessionDB {
 	return session
 }
 
+func CreateAPIKey(key string) {
+	apikeys.AddAPIKey(key)
```

Comment:
See the the function `CreateEventStream` in `services/eventStream.go` for how we generate a random string. We should do that here and then call the persistence method. This should not take in a parameter.